docs/OV_Runtime_UG/multi_device.md

```diff
@@ -1,39 +1,41 @@
 # Running on multiple devices simultaneously {#openvino_docs_OV_UG_Running_on_multiple_devices}
 
-## Introducing the Multi-Device Plugin (C++)
 
-@sphinxdirective
-.. raw:: html
 
-    <div id="switcher-cpp" class="switcher-anchor">C++</div>
-@endsphinxdirective
+The Multi-Device execution mode, or MULTI for short, acts as a "virtual" or a "proxy" device, which does not bind to a specific type of hardware. Instead, it assigns available computing devices to particular inference requests, which are then executed in parallel. 
 
-The Multi-Device plugin automatically assigns inference requests to available computational devices to execute the requests in parallel. By contrast, the Heterogeneous plugin can run different layers on different devices but not in parallel. The potential gains with the Multi-Device plugin are:
+The potential gains from using Multi-Device execution are:
+* improved throughput from using multiple devices at once,
+* increase in performance stability due to multiple devices sharing inference workload.
 
-* Improved throughput from using multiple devices (compared to single-device execution)
-* More consistent performance, since the devices share the inference burden (if one device is too busy, another can take more of the load)
+What is important, the Multi-Device mode does not change the application logic, so it does not require you to explicitly compile the model on every device or create and balance inference requests. From the application point of view, this is just a conventional device that internally handles the actual machinery.
 
-Note that with Multi-Device the application logic is left unchanged, so you don't need to explicitly compile the model on every device, create and balance the inference requests and so on. From the application point of view, this is just another device that handles the actual machinery. The only thing that is required to leverage performance is to provide the multi-device (and hence the underlying devices) with enough inference requests to process. For example, if you were processing 4 cameras on the CPU (with 4 inference requests), it might be desirable to process more cameras (with more requests in flight) to keep CPU and GPU busy via Multi-Device.
+Note that performance increase in this mode comes from utilizing multiple devices at once. This means that you need to provide the devices with enough inference requests to keep them busy or you will not benefit much from using MULTI.
```

Comment:
```suggestion
Note that the performance increase in this mode comes from utilizing multiple devices at once. This means that you need to provide the devices with enough inference requests to keep them busy, otherwise you will not benefit much from using MULTI.
```